cognite/neat/_graph/transformers/_value_type.py

```diff
@@ -64,3 +71,73 @@ def transform(self, graph: Graph) -> None:
                     graph.remove((s, property_uri, o))
                     new_property = URIRef(f"{property_uri}_{remove_namespace_from_uri(value_type_uri)}")
                     graph.add((s, new_property, o))
+
+
```

Comment:
This is useful transformer when one is unable to attach datatype to values when extracting triples from source